src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -20,25 +20,12 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.UnknownHostException;
-import java.util.List;
-
-import org.apache.commons.lang3.StringUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.ericsson.gerrit.plugins.eiffel.configuration.EiffelPluginConfiguration;
-import com.ericsson.gerrit.plugins.eiffel.events.models.Link;
-import com.ericsson.gerrit.plugins.eiffel.exceptions.NoSuchElementException;
-import com.ericsson.gerrit.plugins.eiffel.storage.EventStorage;
-import com.ericsson.gerrit.plugins.eiffel.storage.EventStorageFactory;
 
 /**
  * Base class with common functionality for event generators.
  *
  */
 public class EiffelEventGenerator {
```

Comment:
Should this class be really called EiffelEventGenerator, as the code doesn't contain anything regarding eiffel events at al ?

Reply:
Perhaps the name EiffelEventSourceGenerator is more apt

Reply:
Not my naming but this is the base class for all the generators

Reply:
True, but it doesn't contain any logic pertaining to event generation. It basically extracts information from gerrit.

Reply:
I won't change the name in this pull request. Want to keep it small.